mps_youtube/commands/misc.py

```diff
@@ -104,8 +103,13 @@ def fetch_comments(item):
           'maxResults': 50,
           'part': 'snippet'}
 
-    jsdata = pafy.call_gdata('commentThreads', qs)
-
+    jsdata = None
+    try:
+        jsdata = pafy.call_gdata('commentThreads', qs)
+    except pafy.util.GdataError as e:
+        print(" " * util.getxy()[0] + "\r") # blank the li
+        print(str(e).replace(" identified by the <code><a href=\"/youtube/v3/docs/commentThreads/list#videoId\">videoId</a></code> parameter", ""))
+        return None
```

Comment:
Instead of printing and returning, this should probably re-raise the exception with the new text.